crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/prosecution_concluded/listener/ProsecutionConcludedListenerTest.java

```diff
@@ -97,10 +105,12 @@ void givenJSONMessageIsReceived_whenProsecutionConcludedListenerIsInvoked_thenRe
     @Test
     void givenInvalidMessage_whenProsecutionConcludedListenerIsInvoked_thenShouldNotCallService() {
         String message = getSqsMessagePayload();
-        when(gson.fromJson(message, ProsecutionConcluded.class)).thenThrow(new ValidationException());
+        when(gson.fromJson(message, ProsecutionConcluded.class)).thenThrow(
+            new ValidationException(ProsecutionConcludedValidator.OU_CODE_IS_MISSING));
         prosecutionConcludedListener.receive(message, new MessageHeaders(new HashMap<>()));
         verify(prosecutionConcludedService, times(0)).execute(any());
-
+        verify(deadLetterMessageService, times(1)).logDeadLetterMessage(
+            ProsecutionConcludedValidator.OU_CODE_IS_MISSING, message);
```

Comment:
Don't need to the `verify` call here, if the stub isn't called it will fail regardless. We also need to check that the `MAAT_ID` validation is being called. This should also be replicated in the scheduler tests.

Reply:
Not sure why we don't need the `verify` call here. Added it to check that we are calling to log the message on a `ValidationException` within the listener, I _think_ we do want to check this?

Reply:
You're right, I thought we were stubbing it in the same method for some reason 🙃